package.json

```
@@ -1,12 +1,12 @@
 {
-  "name": "console-monorepo",
+  "name": "console-sdk-monorepo",
   "license": "Apache-2.0",
   "version": "1.0.0",
   "private": true,
   "sideEffects": false,
   "workspaces": [
-    "packages/microfronent-sdk",
-    "packages/vite-helpers-console-microfrontend",
+    "packages/console-sdk-microfrontend",
+    "packages/console-sdk-vite-helpers",
     "packages/webpack-helpers-console-microfrontend"
```

Here, we couldn't set the below?

```suggestion
    "packages/*"
```